lib/tasks/migrate_saphyr_data.rake

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+namespace :migrate_saphyr_data do
+  task create_runs: :environment do |_t|
+    # samples and requests
+    table = CSV.parse(File.read(Rails.root.join('lib', 'data', 'saphyr_samples.csv')), headers: true)
+    table.each do |row|
+      created_at = Date.parse(row['created_at'])
+      sample = Sample.create!(name: row['name'], species: row['species'], external_id: row['external_id'], created_at: created_at)
```

Comment:
This samples table has a external_study_id column, maybe this can be removed if the external_study_id is on the request?